.drone.env

```diff
@@ -1,4 +1,4 @@
 # The test runner source for API tests
-APITESTS_COMMITID=dc3ed28696c6823ad181627a6ac92bbeccf06565
-APITESTS_BRANCH=master
+APITESTS_COMMITID=0b7ddefb4c60408ecc68eef68bc887e73360aec1
```

Comment:
most failure looks like the changes in line number. I have updated the commit of branch `tests/test-coverage-depth-infinity`.
Please, use `1906b5bb3048a33f1a61dc828e4e99b92da9a54d` commit hash